charts/zenko-filer/templates/deployment.yaml

```diff
@@ -0,0 +1,122 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "zenko-filer.fullname" . }}
+  labels:
+    app: {{ template "zenko-filer.name" . }}
+    chart: {{ template "zenko-filer.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "zenko-filer.name" . }}
+      release: {{ .Release.Name }}
+  template:
+    metadata:
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+      labels:
+        app: {{ template "zenko-filer.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Values.nfsd.name }}
+          image: "{{ .Values.nfsd.image.repository }}:{{ .Values.nfsd.image.tag }}"
+          imagePullPolicy: {{ .Values.nfsd.image.pullPolicy }}
+          ports:
+            - name: nfs
+              containerPort: 2049
+          volumeMounts:
+            - name: portmap-data
+              mountPath: /run
+            - name: metadata-data
+              mountPath: /conf
+            - name: nfsd-config
+              mountPath: /template
+          env:
+            - name: CONFIGURATION_DIRECTORY
+              value: "/conf"
+            - name: CONFIGURATION_TEMPLATE
+              value: "/template/nfsd.conf"
+            - name: ZENKO_MD_HOST
+              value: "127.0.0.1"
+            - name: ZENKO_MD_PORT
+              value: "9000"
+            - name: REDIS_SENTINEL_BOOTSTRAP
+              value: "{{- printf "%s-%s:26379" .Release.Name "redis-ha-sentinel" | trunc 63 | trimSuffix "-" -}}"
+            - name: REDIS_CLUSTER_NAME
+              value: "mymaster"
+            - name: NFSD_LOG_FILE
+              value: "/logs/scality-nfsd.log"
+          args: ['scality.nfsd', '-N', 'INFO', '-F', '-f', '/conf/scality-nfsd.conf', '-L', '/dev/stdout']
+          # TODO: livenessProbe:
```

Comment:
livenessProbe and readinessProbe are defined below, comments seem outdated

Reply:
They are defined below for the ```zenko-md``` container there aren't any for the ```nfsd``